### PR TITLE
[MNOE-860] - Display demo organization flag

### PR DESCRIPTION
**Pull request by fgourichon**

### CODE REVIEW

File: src/app/components/mnoe-organizations-list/mnoe-organizations-list.coffee [hunk 1]
```diff
@@ -24,6 +24,7 @@
     # table generation - need to get the locale first
     $translate(
       ["mnoe_admin_panel.dashboard.organization.account_frozen_state",
+      "mnoe_admin_panel.dashboard.organization.demo_account_state",
```
Comment (alexnoox) [inline src/app/components/mnoe-organizations-list/mnoe-organizations-list.coffee]: Do you need to translate `mnoe_admin_panel.dashboard.organization.demo_account_state` here. It is not fetched from `locale` array but passed to the translate `directive`.

**Comment by fgourichon** [comment]
@alexnoox fixed + added description of reason for the account to be demo

**Comment by ouranos** [comment]
Screenshots please?
@alexnoox please review latest changes

**Comment by fgourichon** [comment]
Screenshots below

image
image

**Pull request merged**